packages/Input/src/Input.js

```diff
@@ -108,7 +109,7 @@ const Input = props => {
     ...moreProps
   } = props;
 
-  if (moreProps.value) {
+  if (moreProps.value || moreProps.value === "") {
```

Comment:
`""` as `value` was evaluated as false, and so when a different `value` is provided there would be warning that the input component is being changed from controlled to uncontrolled. 

![image](https://user-images.githubusercontent.com/4040102/81877696-a0677d00-953a-11ea-884f-0acd6afffa6c.png)